src/SignhostAPIClient/Rest/SignHostApiClient.cs

```diff
@@ -26,6 +26,52 @@ public class SignHostApiClient
 		private readonly ISignHostApiClientSettings settings;
 		private readonly HttpClient client;
 
+		public static bool IsPostbackChecksumValid(
+			IDictionary<string, string[]> headers,
+			string body,
+			string sharedSecret,
```

Comment:
I think I'd rather have the shared secret as part of the ISignhostapiclientsettings ... ?